### PR TITLE
Fireflies

### PR DESCRIPTION
**Pull request by PhilipRosedale**
Creates a number of 'fireflies' which are little cubes that fall to surface and bounce around and light up when you are near them.  Entity script based so they are not moving or lighting up when no one is around.  Run 'makeFireflies.js' to make some.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4728eeb862a55fb9a10d234d89cc708ca1fa7b61 available at: https://gist.github.com/eee48cdedd4ed7765264<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7287.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7287.dmg

**Comment by cozza13** [comment]
Tested and they work. 

https://i.gyazo.com/9d0dcc11a4f7401374048202ad3be117.gif

File: examples/fireflies/firefly.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+//
```
Comment (sethalves) [inline examples/fireflies/firefly.js]: This file needs its header/copyright blob

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5230d3a7055e50518ccec8e368311194554de756 available at: https://gist.github.com/eee48cdedd4ed7765264<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7287.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7287.dmg

**Pull request merged**